.gitignore

```diff
@@ -1 +1,2 @@
 vendor
+.idea/
```

Comment:
Please set the settings in PHPStorm so that it adds a blank line at end of files. and also this should be in your global `gitignore`. Do not think this needs to be added here. [Reference](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)